src/Resources/parser/netkit/DepParser.py

```diff
@@ -22,10 +22,15 @@ def parse(path):
         line_number = 1
         line = dep_mem_file.readline().decode('utf-8')
         while line:
+            line = line.strip()
+            if len(line) == 0:
```

Comment:
This is good, however it can be squeezed into one line:
```python
if not line or line.startswith('#'):
    continue
```

Reply:
Yeah that is probably the best way of doing it